packages/TelegramClient-Core.package/TCCCore.class/instance/handleUserEvent..st

```diff
@@ -0,0 +1,8 @@
+events
+handleUserEvent: anEvent
+	anEvent at: '@extra'  ifPresent: [:extra|
+		(self pendingMessages at: extra) value: anEvent. 
+	]
+	ifAbsent: [
+		self halt.
+	]
```

Comment:
Also, deleting the block from the dictionary would be a good idea

Reply:
It is a remnant from earlier versions where `getMe` did not include an `@extra`. Will fix both.

Reply:
> Assume @extra will be always present and create a test to validate this hypothesis.

Do we really need a test for something that is clearly stated in the API reference?

Reply:
As Prof. Hirschfeld said, we should specify the requirements we have for the API with assertions. I think this can be tested in the test for adding/resolving pendingMessages.